lib/config/cli.spec.ts

```diff
@@ -103,6 +103,35 @@ describe('config/cli', () => {
         hostRules: [],
       });
     });
+    it('parses json lists correctly as a list of packageRules', () => {
+      argv.push(
+        `--package-rules=[{"excludePackageNames":["react"],"enabled":false},{"matchPackageNames":["angular"],"rangeStrategy":"pin"}]`
```

Comment:
Can you test that this actually works via CLI? I think you might need `'` wrapping it